components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.h

```diff
@@ -50,4 +91,5 @@
 + (void)applyColorScheme:(nonnull id<MDCColorScheme>)colorScheme
```

Comment:
Add comments to the old APIs mapping to the new APIs that should be used instead.

Reply:
Done.